charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
+
+## Description
+
+The Data Coordination Platform (DCP) PM team is responsible for the oversight of the DCP *as product*, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+The core team is composed of *Project Leads* and *Product Owners* from each DCP project, *DCP UX* (Voice of the Customer), and a *Science Program Manager*.
+
+## Definitions
+
+* A **Thematic Roadmap** identifies and expresses customer needs as a small collection of themes or strategic objectives which:
+    * Focuses the DCP community on a documented set of high-level customer priorities
+    * Excites users about the future of DCP
+
+* **Request for Comments (RFC)** is the transparent process that manages all significant enhancements to the DCP software technical and governance documentation, both normative and informative
+
+## In-scope
+
+*Governance* 
+
+* Develop standard processes and best practices for DCP community governance including (but not limited to):
+    * Program and technical oversight, decision making, and conflict resolution
+    * Charters for DCP projects
+    * Enhancements (RFC) to DCP software or governance 
+    * Planning
+    * Communication
+
+* Review and approve non-technical charters for DCP projects
+
+* Act as final arbiter for escalated conflicts from the DCP community
```

Comment:
I might include mention of technical charters/RFC arbitration... when technical charters/RFC ultimately propose something that doesn't work for Project Leads, Product Owners from each DCP project, DCP UX (Voice of the Customer), and the Science Program Managers what do we do?

Reply:
@briandoconnor - do you have any proposals to share? Even the IETF is a bit ambiguous on this point:

https://tools.ietf.org/html/rfc2026#section-6.5.1
_The IAB shall then review the situation and attempt to resolve it in a manner of its own choosing_

Reply:
Ultimately I think if there's a problem in a project that requires arbitration between parties it should be the project lead that mitigates and manages this.  That's my personal opinion.